about.html

```diff
@@ -0,0 +1,112 @@
+        <p class="about__hero-contact">Please contact us via email for further questions about IWES conference 2023!</p>
+        <p class="about__hero-email"><a href="#">womenconference@IWES.org</a></p>
+    </div>
+ </header>
+ <main>
+<section class="decide">
+<div class="decide__container">
+    <h2 class="decide__container-title">International Women Empowerment </h2>
+    <h3 class="decide__container-sub-title">International Women Empowerment </h3>
+    <p class="decide__container-body">The logo of IWES 2023 was decided through Competition from 1st Jan. - 31st Jan. 2023. </p>
+    <div class="decide__container-container">
+        <img src="asset/images/logoicon.png" alt="logo-icon" class="container-logo-icon">
+        <div class="decide__container-container-details">
+            <h2 class="decide__container-container-details-text">IWES INTERNATIONAL CONFERENCE 2023 </h2>
+            <p class="decide__container-container-details-body">14th - 16th Sep 2023 | Abuja, Nigeria </p>
+        </div>
+    </div>
+</div>
+</section>
+
+<section class="previous">
+    <div class="previous__container">
+        <h4 class="previous__container-header">
+            See the Previous IWES Conferences 
+        </h4>
+        <p class="previous__container-body">
+            Take a look at the last two conferences which took place in California and Budapest. 
+        </p>
+        <div class="previous__container-images">
+            <div class="previous__container-images-card">
+                <img src="asset/images/bg1.png" alt="evebt 1" class="previous__container-images-card-image">
+                <div class="previous__container-images-card-details">
+                   <div class="previous-text">
+                    <h2 class="previous__container-images-card-details-header">2021</h2>
+                    <p class="previous__container-images-card-details-body">IMES conference in Moscow,Russia</p>
+                   </div>
+                </div>
+            </div>
+
+            <div class="previous__container-images-card">
+                <img src="asset/images/bg2.png" alt="evebt 1" class="previous__container-images-card-image">
+                <div class="previous__container-images-card-details">
+                    <div class="previous-text">
+                    <h2 class="previous__container-images-card-details-header">2022</h2>
+                    <p class="previous__container-images-card-details-body">IMES conference in Stockholm, Sweden</p>
+                </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+</main>
+
+
+    <footer>
+        <div class="footer__container">
+            <img src="asset/images/logowhite.png" alt="footer-logo" class="footer-logo">
+        <p class="about-reserve">2023 IMES. All rights reserved.</p>
+        </div>
+    </footer>
```

Comment:
Please update the footer in your about page to have a dark background so it is aligned with the behance design🙏

Behance design
<img width="554" alt="image" src="https://user-images.githubusercontent.com/37116322/232935545-9df19803-8a58-415b-93db-866cffba5b12.png">